### DISCUSSION TITLE
How to display Site logo in Search Results

### DISCUSSION THREAD

**Discussion by szyvett** [microsoft-search/pnp-modern-search#2037] [Q&A] [answered]
Hi all, 

I was wondering if someone could help me with an issue I am facing. I need to display certain site(s) in my PnP Search Results web part (PnP Search v4) and I need to show the site logo instead of the image for SharePoint sites (as seen below):

![chrome_I119wITjkW](https://user-images.githubusercontent.com/32570227/165294534-66227409-70af-48a8-a945-bce25a8f165a.png)

I have been looking at slots to try and figure out how to display the site logo instead of the file type (@root.slots.FileType), but I haven't found any of the default slots helpful nor could I create a new slot from the list in layouts (as I have seen from the [documentation](https://microsoft-search.github.io/pnp-modern-search/usage/search-results/slots/#define-your-custom-slots)). 

Can someone please help how I can display the [logo] [SiteName] for each result of my search? 

Many thanks, 
Yvett

**Comment by FranckyC** [comment, marked as answer]
Hi @szyvett, did you add the 'SiteLogo' property to the selected properties in the SharePoint data source? By adding it you should see the site logo (if configured in the site) by default with the 'List' layout:

![image](https://user-images.githubusercontent.com/5659122/165623637-9a6c5475-1733-4341-a716-f403e0982cc8.png)

**Reply by szyvett** [reply]
Sorry for the late reply, thank you for your help @FranckyC
